### ISSUE TITLE
Avoid blinking effects

### ISSUE THREAD

**Issue by davorpa** [Adalab/modulo-3-evaluacion-final-sorayav#5]
Hi @sorayav @adalab-tech 

At the point of view of the www accessibility the blinking effects should be avoid. Infinite shaking or blinking shapes / texts can cause epilepsy.

![image](https://user-images.githubusercontent.com/3125580/92323929-e6da6d80-f03c-11ea-97d8-f455356a8a6c.png)

Seen at

https://github.com/Adalab/modulo-3-evaluacion-final-sorayav/blob/39dfee8dd9b95799d1ca424b05caf1d9d56ad0f7/src/scss/layouts/_header.scss#L36-L85

and also 

https://github.com/Adalab/modulo-3-evaluacion-final-sorayav/blob/39dfee8dd9b95799d1ca424b05caf1d9d56ad0f7/src/scss/layouts/_notfound.scss#L44-L78

**Comment by davorpa** [comment]
Seen at

https://github.com/Adalab/modulo-3-evaluacion-final-sorayav/blob/39dfee8dd9b95799d1ca424b05caf1d9d56ad0f7/src/scss/layouts/_header.scss#L36-L85

**Comment by sorayav** [comment]
Your feedback hasn't been requested and is not welcome. Please refrain from trying to add me to LinkedIn or other networks as part of your usual harassment to women.
--

Tu feedback no ha sido pedido y no es bien recibido. Por favor abstente de intentar agregarme a LinkedIn u otras redes como parte de tu acoso usual a mujeres.

**Comment by davorpa** [comment]
Are you anoying me? I'm not doing nothing of that

I'm only try to contribute to a opensource project that hasn't CONTRIBUTING.MD, so anybody can contribute.

The fact it's I'm epileptic person and you are discriminating me by my disability closing my request.

Also if you know administration/educacion websites needs be builded with at least AA accesibility passing test. Not it's @adalab-tech a education community?

Read a little bit about https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-three-times.html
